lwallet/Pos/Wallet/Light/Web/Server.hs

```diff
@@ -1,24 +0,0 @@
-{-# LANGUAGE DataKinds     #-}
-{-# LANGUAGE TypeOperators #-}
-
--- | Module for lite-wallet implementation of Daedalus API. We don't
--- maintain it because we don't know whether we will ever use at least
--- 30% of its current state.
-
-module Pos.Wallet.Light.Web.Server
-       ( walletServeWebLite
-       ) where
-
-import           Universum
-
-import           Pos.Communication.Protocol (SendActions)
-import           Pos.Wallet.Light.Mode      (LightWalletMode)
-
-walletServeWebLite
```

Comment:
There is such code in `Main.hs`:
```
                Serve __webPort __webDaedalusDbPath -> error "light wallet server is disabled"
                -- Serve webPort webDaedalusDbPath -> worker walletServerOuts $ \sendActions ->
                --     walletServeWebLite sendActions webDaedalusDbPath False webPort
```

I think it makes sense to delete it (and `Serve` command) as well.